web_sdk/BUILD.gn

```diff
@@ -52,6 +51,30 @@ copy("web_ui_library") {
   ]
 }
 
+prebuilt_dart_action("generate_flutter_web_summary") {
```

Comment:
Not sure about the name, does it only generate summary or does it also generate the JS SDK?